meinberlin/apps/budgeting/api.py

```diff
@@ -8,11 +11,45 @@
 from .serializers import ProposalSerializer
 
 
+# To be changed to a more general IdeaPagination, when using
+# pagination via rest api for more idea lists
+class BudgetPagination(PageNumberPagination):
+    page_size = 15
+
+    def get_next_page(self):
+        if self.page.has_next():
+            return self.page.next_page_number()
+        else:
+            return None
+
+    def get_previous_page(self):
+        if self.page.has_previous():
```

Comment:
Why don't you use the `next` and `previous`, the serializer just gives you for free?